src/Dispatcher.php

```diff
@@ -177,7 +177,11 @@ public function handleError(\Exception $errors)
 
         $action_result = "";
 
-        $logger->error('Error occurred. ', ['error' => get_class($errors), 'message' => $errors->getMessage()]);
+        $logger->error('Error occurred. ', [
+            'error'     => get_class($errors),
+            'message'   => $errors->getMessage(),
+            'trace'      => $errors->getTrace(),
```

Comment:
I think it is better to replace  `$errors->getTrace()` to `$errors->getTraceAsString()` since `getTrace` contains too many informations for the log.